### DISCUSSION TITLE
Can I get TextAlign?

### DISCUSSION THREAD

**Discussion by patsornk** [ueberdosis/tiptap#2098] [General Questions] [answered]
Can I get TextAlign that is left, center or right? on console log or something. When I set TextAlign right my floating menu overflow in textbox. It make a horizontal scroll like this.
![image](https://user-images.githubusercontent.com/76081944/139175333-890f5c3f-dcbd-4b21-a466-0b01d5636c4b.png)

![image](https://user-images.githubusercontent.com/76081944/139175324-c616698d-2679-40ca-9ba3-3a673cc2fd99.png)

**Comment by philippkuehn** [comment, marked as answer]
you can get the current textalign:

```js
editor.isActive({ textAlign: 'right' })
```

**Reply by patsornk** [reply]
It's worked. Thank you.
